### DISCUSSION TITLE
Wayland Support

### DISCUSSION THREAD

**Discussion by Frereneo** [rustdesk/rustdesk#3293] [Q&A] [answered]
Hello,
First of all, I would like to congratulate you for your project. It's really great !
I have a small question about Wayland:
Is it planned to be able to connect one day from a post client to a machine that works with the wayland protocol instead of X11?
If yes, do you know when it will be implemented?
Thanks in advance for your answer.
Sincerely,

**Comment by neoktgc** [comment]
Rustdesk is a great software. Congratulations.
With Wayland you are close.
I think there is not much missing.
I think it's a shame that the Wayland developers don't support the connection!

**Reply by rustdesk** [reply]
it is closed because it is finished and supported in our nightly build. it is a shame you do not read it carefully.

**Reply by neoktgc** [reply]
Sorry

**Comment by rustdesk** [comment, marked as answer]
Supported experimentally in nightly build.

**Reply by Frereneo** [reply]
Hello,
Sorry, I thought it was only about a takeover from a Wayland computer, not that it was also about a Wayland target computer.
Thanks for your help. And sorry again.
Sincerely,

**Comment by secretmango** [comment]
Can you specify what nightly build that was and if that has now landed in stable, for example the Flatpak?

**Reply by FurtadoPires** [reply]
Version 1.2.4 has wayland support.
